docs/reference_guide.md

```diff
@@ -2701,6 +2702,28 @@ contains the memcopy'd value so the value at `print()` invocation time will be
 printed.  However for maps, only the handle to the map is queued up, so the
 printed map may be different than the map at `print()` invocation.
 
+## 24. `strftime()`: Formatted timestamp
+
```

Comment:
Wonder if we should mark this experimental as we're so close to the 0.11 release What do you think @danobi